ui/src/app/views/concept-add-modal/component.ts

```diff
@@ -114,12 +115,17 @@ export class ConceptAddModalComponent {
       return;
     }
 
-    this.conceptSetsService.createConceptSet(this.wsNamespace, this.wsId, {
-          name: this.name,
-          description: this.description,
-          domain: this.selectDomain,
-          concepts: this.selectConceptList
-        })
+    const conceptSet: ConceptSet = {
+      name: this.name,
+      description: this.description,
+      domain: this.selectDomain
+    };
+    const request: CreateConceptSetRequest = {
+      conceptSet: conceptSet,
+      addedIds: conceptIds
+    };
+
+    this.conceptSetsService.createConceptSet(this.wsNamespace, this.wsId, request)
```

Comment:
Error handling:  The error message for bad request is set to conflicting name, however that will not be true always. So we might want to set the value of variable error message as that of  the message attribute of error object

Reply:
The two new cases for throwing BadRequestException in the code -- domain mismatch or too many concepts -- should never occur in the UI as written.  So I'm actually not sure we need to worry about this.